src/components/Location/Heading.css

```diff
@@ -1,12 +1,11 @@
-.heading {
-  margin-bottom: 30px;
-  text-align: center;
-  vertical-align: middle;
-}
-
 .logo {
   display: block;
   width: auto;
   height: 70px;
   margin: auto;
 }
+
+.caption {
+  margin: 0 0 30px;
+  vertical-align: middle;
```

Comment:
You can remove the `vertical-align: middle;` (vertical-align only has effect on inline elements (h1 isn't inline) or tables or similar) and give some top margin, so the caption doesnt stick to the image too much, like `margin: 20px 0 30px;`
(Although having a special margin here means, we still don't have uniform styling for the captions everywhere, but oh well :D )